backend/mcp_backend/integration/crew_mcp_integration.py

```
@@ -0,0 +1,634 @@
+
+        Args:
+            query: The query to process
+
+        Returns:
+            Combined response from multiple crews
+        """
+        logger.info(f"Starting multi-server CrewAI integration for query: {query}")
+
+        available_servers = self.get_available_servers()
+        if not available_servers:
+            raise FileNotFoundError("No MCP servers found")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)